### PR TITLE
Sndfile only

### PR DESCRIPTION
**Pull request by eblanca**
This PR removes audioin_raw.c and uses audioin_sndfile.c for any input file, even raw.
Works also in cases of wrong endian samples (little<->big), just run with "-x".

### CODE REVIEW

**Comment by eblanca** [comment]
I see this solution does not handle the case of input through `stdin`, so I'll do some further tests.

**Comment by eblanca** [comment]
I think it is fixed now.

**Comment by njh** [comment]
Thanks, I will take a look.

There should be a test for reading from STDIN.

One of the reasons for having audioin_raw.c, was so that it was possible to use twolame on systems that don't have/support libsndfile. I am not sure how many people actually use this feature though, as libsndfile is very widely supported.

**Comment by eblanca** [comment]
> it was possible to use twolame on systems that don't have/support libsndfile

The `configure` script currently does not build the frontend when libsndfile is not detected.

File: frontend/frontend.c [hunk 1]
```diff
@@ -656,15 +697,12 @@ int main(int argc, char **argv)
...
-    // Calculate the size and number of frames we are going to encode
-    if (sfinfo.frames)
-        total_frames = (sfinfo.frames -1) / TWOLAME_SAMPLES_PER_FRAME +1;
-
 
     // Now do the reading/encoding/writing
     while ((samples_read = inputfile->read(inputfile, pcmaudio, audioReadSize)) > 0) {
         int bytes_out = 0;
 
+#if 0
```
Comment (njh) [inline frontend/frontend.c]: Can this be deleted instead of the code being disabled?

Comment (eblanca) [reply frontend/frontend.c]: Oh, of course yes, it can! I forgot to delete that bunch of statements.

**Comment by njh** [comment]
Merge conflicts resolved:
https://github.com/njh/twolame/tree/sndfile_only

**Comment by njh** [comment]
Build passing:
https://travis-ci.org/njh/twolame/builds/309061051

**Comment by njh** [comment]
This is now merged into master.

As there is now only a single input method, I removed the audioin shim and audioin_sndfile.c.

**Pull request closed without merging**